static/js/admin.js

```diff
@@ -792,7 +792,7 @@ function _setup_page() {
         e.stopPropagation();
         var emoji_status = $('#admin-emoji-status');
         var emoji = {};
-        $(this).serializeArray().map(function (x) {emoji[x.name] = x.value;});
+        $(this).serializeArray().map(function (x) {emoji[x.name] = x.value; return emoji;});
```

Comment:
So this kind of precedes your change, but the use of `map` here doesn't really make sense.  We should instead do something like this:

```
_.each($(this).serializeArray() function (obj) {
    emoji[obj.name] = obj.value;
});
```